drone/repo_info.go

```diff
@@ -39,7 +43,14 @@ func repoInfo(c *cli.Context) error {
 
 	repo, err := client.Repo(owner, name)
 	if err != nil {
-		return err
+		re := regexp.MustCompile("client error \\d{3}")
+		err_msg := re.FindAllString(err.Error(), -1)
+		if err_msg != nil {
+			r := regexp.MustCompile("\\d{3}")
+			status_code, _ := strconv.Atoi((r.FindAllString(err_msg[0], -1))[0])
+			status_text := http.StatusText(status_code)
+			return fmt.Errorf("%s: %s", err_msg[0], status_text)
+		}
```

Comment:
This can be removed. We are migrating to a single page application and therefore will no longer be returning html error pages, and will instead return a 403 error with an error message.